assets/i18n/tr.json

```diff
@@ -0,0 +1,299 @@
+  "I18N_LIBRARY_NO_EXPLORATION_GROUPS": "Gösterilecek konu anlatım grubu bulunamadı.",
+  "I18N_LIBRARY_NO_OBJECTIVE": "Hedef belirtilmedi.",
+  "I18N_LIBRARY_N_CATEGORIES": "{categoriesCount, plural, =1{1 Kategori} other{# Kategori}}",
+  "I18N_LIBRARY_N_LANGUAGES": "{languagesCount, plural, =1{1 Dil} other{# Dil}}",
+  "I18N_LIBRARY_PAGE_TITLE": "Konu Anlatımı Kütüphanesi - Oppia",
+  "I18N_LIBRARY_RATINGS_TOOLTIP": "Oylamalar",
+  "I18N_LIBRARY_SEARCH_PLACEHOLDER": "Hangi konu hakkında merak duyuyorsun?",
+  "I18N_LIBRARY_SUB_HEADER": "Konu anlatımlarını inceleyerek macerana başla.",
+  "I18N_LIBRARY_UPDATED": "GÜNCELLENMİŞ",
+  "I18N_LIBRARY_VIEWS_TOOLTIP": "Görüntülenme",
+  "I18N_LIBRARY_VIEW_ALL": "Hepsini görüntüle",
+  "I18N_PLAYER_BACK": "Geri",
+  "I18N_PLAYER_CARD_NUMBER_TOOLTIP": "Kart #",
+  "I18N_PLAYER_COMMUNITY_EDITABLE_TOOLTIP": "Birlikte düzenlenebilir",
+  "I18N_PLAYER_CONTINUE_BUTTON": "İlerle",
+  "I18N_PLAYER_CONTINUE_NEXT_CARD": "Bir sonraki karta ilerle",
+  "I18N_PLAYER_CONTRIBUTORS_TOOLTIP": "Katılımcılar",
+  "I18N_PLAYER_EDIT_TOOLTIP": "Düzenle",
+  "I18N_PLAYER_EMBED_TOOLTIP": "Gömülü",
+  "I18N_PLAYER_FEEDBACK_TOOLTIP": "Dönüt",
+  "I18N_PLAYER_FORWARD": "İlerle",
+  "I18N_PLAYER_INFO_TOOLTIP": "Bilgi",
+  "I18N_PLAYER_LAST_UPDATED_TOOLTIP": "Son güncellenmiş",
+  "I18N_PLAYER_LEAVE_FEEDBACK": "Yazarlar için dönüt bırak...",
+  "I18N_PLAYER_LOADING": "Yükleniyor...",
+  "I18N_PLAYER_NO_OBJECTIVE": "Hedef belirtilmedi.",
+  "I18N_PLAYER_NO_TAGS": "Etiketler belirtilmedi.",
+  "I18N_PLAYER_PLUS_TAGS": "<[additionalTagNumber]>+ daha fazla etiket",
+  "I18N_PLAYER_PREVIOUS_RESPONSES": "Önceki cevaplar (<[previousResponses]>)",
+  "I18N_PLAYER_RATE_EXPLORATION": "Yeni bir şeyler öğrendin mi? Bu konu anlatımını nasıl oylardın?",
+  "I18N_PLAYER_RATINGS_TOOLTIP": "Oylamalar",
+  "I18N_PLAYER_RECOMMEND_EXPLORATIONS": "Sonraki önerilen:",
+  "I18N_PLAYER_REPORT_MODAL_BODY_AD": "Yanıltıcı içerik, mesaj ya da reklam",
+  "I18N_PLAYER_REPORT_MODAL_BODY_ADDITIONAL_DETAILS": "Lütfen moderatörler için ek bilgi sağlayın:",
+  "I18N_PLAYER_REPORT_MODAL_BODY_HEADER": "Sorun nedir?",
+  "I18N_PLAYER_REPORT_MODAL_BODY_OTHER": "Diğer",
+  "I18N_PLAYER_REPORT_MODAL_BODY_POOR_EXPERIENCE": "Zayıf öğrenme deneyimi",
+  "I18N_PLAYER_REPORT_MODAL_FOOTER_CANCEL": "İptal",
+  "I18N_PLAYER_REPORT_MODAL_FOOTER_SUBMIT": "Gönder",
+  "I18N_PLAYER_REPORT_MODAL_HEADER": "Bu konu anlatımını rapor et",
+  "I18N_PLAYER_REPORT_SUCCESS_MODAL_BODY": "Raporun değerlendirilmek üzere moderatörlere gönderilmiştir.",
+  "I18N_PLAYER_REPORT_SUCCESS_MODAL_CLOSE": "Kapat",
+  "I18N_PLAYER_REPORT_SUCCESS_MODAL_HEADER": "Teşekkürler!",
+  "I18N_PLAYER_REPORT_TOOLTIP": "Konu Anlatımını Rapor Et",
+  "I18N_PLAYER_RETURN_TO_COLLECTION": "<[collectionTitle]>'a dön",
+  "I18N_PLAYER_RETURN_TO_EDITOR": "Editöre dön",
+  "I18N_PLAYER_RETURN_TO_LIBRARY": "Kütüphaneye dön",
```

Comment:
Ah, actually -- we recently made a small change to the translations. Please see https://github.com/oppia/oppia/blob/develop/assets/i18n/en.json#L171 -- RETURN_TO_LIBRARY has now changed to GO_TO_LIBRARY, since we realized that the player doesn't always enter an exploration from the library (they may access the link directly, or via the author's profile page, etc.) Could you please amend this? Make sure you put the new translation in the correct place alphabetically.